common/decisions/wc_thorium_decisions.txt

```diff
@@ -0,0 +1,63 @@
+﻿# Warcraft
```

Comment:
You don't need to have `# Warcraft` in files starting with `wc_`, only in files that are from vanilla